tripl/tripl.py

```diff
@@ -180,13 +184,15 @@ def __getitem__(self, key):
         else:
             return results
 
-
     def get(self, key, default=None):
         """Get a list of values corresponding to the given key (attribute).
         
         Note: Eventually we may treat schema in such a way where get returns the singular value for things
         marked cardinality one."""
-        return self[key]
+        try:
+            return self[key]
+        except KeyError:
+            return default
```

Comment:
PyCharm noticed that `default` was never used. I changed functionality to what was probably intended.

Reply:
I think you're right about that.